src/lib/redux/inventoryDataSlice.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+import { InventoryRecord, InventoryUpdateRecord, PurchaseRequestRecord } from '../airtable/interface';
+
+interface inventoryDataSliceState {
+  isLoading: boolean;
+  products: any[];
+  siteInventory: InventoryRecord[];
```

Comment:
We might want to do some restructuring later on how the data is stored. I think we want to easily be able to access the data for each site, so storing within a map based on `siteId` might be a good call. I think this is generally a problem with our current `siteData` structure also, which I need to fix.

The inefficiency I see with this is when we modify the redux store, we'd have to do a deep copy of all inventory (products, purchase requests, etc...) across all the sites in the redux store when we should only really need to make a deep copy of the inventory in the site where we made the inventory modification.

Again, not a huge deal right now and we might be able to do a data restructure some time (maybe spring break) if you wanna put it off until then. Sorry for putting off the `customerData` refactor  so late so you didn't have a reference

Reply:
{ SiteIdA : {
purchaseRequests,
...
},
SiteIdB: {
purchaseRequests,
...
}

Products: []
}

In siteDataSlice: {

currentSiteId: SiteIdB
}